subsquid/schema.graphql

```diff
@@ -92,6 +92,9 @@ type Schedule {
 
 type VestingSchedule @entity {
   id: ID!
+  "account that initiates the schedule"
+  from: String!
```

Comment:
Given that we are adding `from` we can rename `beneficiary` to `to` to keep it consistent